lang/de/lang.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+return [
+    'plugin' => [
+        'name' => 'Lokalisierung',
```

Comment:
I am changing the name to "YAMLocalizer" in the next update, can you translate this? I guess it should use the english version, what do you think?

Reply:
Okay so then it should stay YAMLocalizer Product names are not translated in general!